lib/handler/handler.go

```diff
@@ -133,29 +133,44 @@ func MainHandler(w http.ResponseWriter, r *http.Request, conf *configure.Conf, l
 	case "jpeg":
 		if q.UseWebP() {
 			err = imageprocessor.EncodeWebP(w, img.GetImg(), q.Quality(), false)
+		} else if q.UseAVIF() {
+			w.Header().Set("Content-Type", "image/avif")
```

Comment:
Browser downloads the response body as a file if the explicit header specification is none.